packages/matrix/playwright.config.ts

```diff
@@ -1,4 +1,10 @@
 import { defineConfig, devices } from '@playwright/test';
+import { readdirSync } from 'fs';
+
+let tests = readdirSync('./tests');
+let middle = Math.floor(tests.length / 2);
+let group1 = tests.slice(0, middle);
+let group2 = tests.slice(middle);
```

Comment:
Does it matter that these groups will be unstable if a new test is added in the middle? It looks like the snapshots are duplicated for both groups and `all` so maybe not, just wanted to confirm.

Reply:
yeah, that's why i added snapshots for all groups. so it should be ok that some test modules may move from group to group as more test modules are added